go/wallet_client/main.go

```diff
@@ -115,7 +117,13 @@ flags
 	}
 }
 
-// createMetaData creates the metadata for an outgoing request based on the user.
+const (
+	affinityMDKey   = "session_id"
+	affinityMDValue = "1234"
+)
```

Comment:
Would we ever want these to be flags instead of consts?

Reply:
The key would just be a const I believe.
The value we may want to change later. But we don't need it now.

Reply:
Ack.